pkg/registry/servicecatalog/broker/strategy.go

```diff
@@ -111,7 +111,7 @@ func (brokerRESTStrategy) PrepareForUpdate(ctx kapi.Context, new, old runtime.Ob
 	if !ok {
 		glog.Fatal("received a non-broker object to update from")
 	}
-	newBroker.Spec = oldBroker.Spec
+
```

Comment:
turns out you can't update a thing if you immediately overwrite it with it's old contents.